acapy_agent/wallet/routes.py

```diff
@@ -312,7 +322,14 @@ class DIDListQueryStringSchema(OpenAPISchema):
     method = fields.Str(
         required=False,
         validate=validate.OneOf(
-            [KEY.method_name, SOV.method_name, PEER2.method_name, PEER4.method_name]
+            [
+                KEY.method_name,
+                SOV.method_name,
+                TDW.method_name,
+                CHEQD.method_name,
+                PEER2.method_name,
+                PEER4.method_name,
+            ]
```

Comment:
To enable plugged in methods to be queried, we should loosen this validation, probably just removing the `validate.OneOf` altogether.

Reply:
Removed it.